docker_compose/compose.yaml

```diff
@@ -0,0 +1,52 @@
+# To start, run:
+# % docker compose up
+#
+# What you should see:
+#   1. lots of waiting while containers are downloaded (or nothing if cached)
```

Comment:
A few minutes or go get a cup of coffee kind of waiting?  Can we say a few minutes?

Reply:
It depends on how fast your network is. When Mehul was getting 1MB/s at the conference it would be about 1-1.5 hours (4-5GB of downloads).  Updated comment.